label_studio/tasks/migrations/0032_annotation_updated_by_fill.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 3.2.16 on 2022-11-18 23:38
+
+from core.models import AsyncMigrationStatus
+from core.redis import start_job_async_or_sync
+from django.conf import settings
+from django.db import migrations, models
+from django.db.models import F
+
+from projects.models import Project
+from tasks.models import Annotation
+
+
+def _fill_annotations_updated_by(project):
+    Annotation.objects.filter(project=project).update(updated_by=F('completed_by'))
```

Comment:
I guess the most correct value for updated by is stored in AnnotationHistory, so probably we can use something like:

```
from annotation_history.models import AnnotationHistory as AH

subquery = AnnotationHistory.objects.filter(annotation_id=OuterRef('pk'), action_type=AH.ActionType.UPDATED).order_by('id')[0])

Annotation.objects
  .filter(project=project).
  .annotate(tmp=subquery)
  .update(updated_by=F('tmp__created_by'))
```
or if postgres doesn't support this, we can rewrite it to simple form with bulk_update.